invenio_app_rdm/config.py

```diff
@@ -994,6 +994,25 @@ def github_link_render(record):
 APP_RDM_ADMIN_EMAIL_RECIPIENT = "info@inveniosoftware.org"
 """Admin e-mail"""
 
+APP_RDM_IDENTIFIER_SCHEMES_UI = {
+    "orcid": {
+        "url_prefix": "http://orcid.org/",
```

Comment:
```suggestion
        "url_prefix": "https://orcid.org/",
```

shelve: although I think we need see how we can maybe take advantage of the existing data to get the URL, and avoid Jinja shenanigans